hack/tests/e2e-helm-new.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
```

Comment:
Lets not call this "new", since that will not make any sense very soon. Instead, lets rename the old file to "legacy"

Reply:
Hi @asmacdo, 

These tests are just to ensure implementation of the helm plugin. See that idea is to remove all tests shell scripts for a Golang implementation that was already started before. After we merge this one I will push a PR with the Go tests and removing both scripts. So, because of this IMO shows fine these names. However, if you see that it would be a blocker for we move forward then we can do that.

Reply:
See: (Blocked by this one) Replace tests via shell for Golang WIP #3423

Reply:
Now we have just once which address your concern as well.